modules/installation-guide/partials/proc_configuring-communication-between-che-components.adoc

```diff
@@ -0,0 +1,62 @@
+[id="configuring-communication-between-{prod-id-short}-components_{context}"]
+= Configuring communication between {prod-short} components
+
```

Comment:
Open with an abstract that summarizes the purpose of this topic:

"Depending on how you have deployed Che, select whether Che components communicate by using the internal network or external Kubernetes Ingress or OpenShift Route."